pennylane_qiskit/qiskit_device.py

```diff
@@ -147,22 +170,28 @@ def __init__(self, wires, provider, backend, shots=1024, **kwargs):
         # Initialize inner state
         self.reset()
 
-        # determine if backend supports backend options and noise models,
-        # and properly put together backend run arguments
-        s = inspect.signature(b.run)
-        self.run_args = {}
         self.compile_backend = None
-
         if "compile_backend" in kwargs:
             self.compile_backend = kwargs.pop("compile_backend")
 
+        aer_provider = str(provider) == "AerProvider"
+        self.noise_model = None
         if "noise_model" in kwargs:
-            if "noise_model" in s.parameters:
-                self.run_args["noise_model"] = kwargs.pop("noise_model")
-            else:
+            if not aer_provider or backend != "qasm_simulator":
                 raise ValueError("Backend {} does not support noisy simulations".format(backend))
 
-        if "backend_options" in s.parameters:
+            self.noise_model = kwargs.pop("noise_model")
+
+        # Get further arguments for run
+        s = inspect.signature(b.run)
+        self.run_args = {}
+
+        if aer_provider:
+            # Consider the remaining kwargs as keyword arguments to run
+            self.run_args.update(kwargs)
```

Comment:
`backend_options` seems to be deprecated for the `run` method of `Aer` backends (e.g. for [QasmSimulator](https://qiskit.org/documentation/stubs/qiskit.providers.aer.QasmSimulator.run.html#qiskit.providers.aer.QasmSimulator.run))